pkg/filters/filters.go

```diff
@@ -127,5 +127,17 @@ func GetPolicyName(event *v1.Event) string {
 	if !ok {
 		return ""
 	}
-	return helpers.ResponseGetProcessKprobe(response).GetPolicyName()
+
+	switch ev := (response.Event).(type) {
+	case *tetragon.GetEventsResponse_ProcessKprobe:
+		return ev.ProcessKprobe.PolicyName
```

Comment:
Any reason for not using `GetPolicyName()` here? It's a bit safer if the pointer is `nil` which shouldn't happen normally but better to avoid crashing

Reply:
I think using `GetPolicyName()` is better, thanks!